pkg/envoy/rds/new_response.go

```diff
@@ -0,0 +1,41 @@
+package rds
+
+import (
+	xds_discovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
+	"github.com/golang/protobuf/ptypes"
+
+	"github.com/openservicemesh/osm/pkg/catalog"
+	cat "github.com/openservicemesh/osm/pkg/catalog"
+	"github.com/openservicemesh/osm/pkg/envoy"
+	"github.com/openservicemesh/osm/pkg/envoy/route"
+)
+
+func newResponse(catalog catalog.MeshCataloger, proxy *envoy.Proxy) (*xds_discovery.DiscoveryResponse, error) {
+	proxyIdentity, err := cat.GetServiceAccountFromProxyCertificate(proxy.GetCertificateCommonName())
+	if err != nil {
+		log.Error().Err(err).Msgf("Error looking up Service Account for Envoy with CN=%q", proxy.GetCertificateCommonName())
```

Comment:
Based on the latest changes in logs for audit, we no longer want to log the certificate common name, we should be logging the serial number

Reply:
updated. thank you